package.json

```diff
@@ -17,7 +19,7 @@
     "cors": "^2.8.1",
     "express": "^4.14.1",
     "fh-js-sdk": "^2.18.2",
-    "fh-mbaas-api": "^6.1.5",
+    "fh-mbaas-api": "feedhenry/fh-mbaas-api#FH-3123-sync-horizontal-scaling",
```

Comment:
Is the plan to keep this branch here until it's merged into master and do these tests fail against master?

If so, we should probably disable these tests running by default against `fh-mbaas-api` master.
https://github.com/feedhenry/fh-mbaas-api/blob/master/.travis.yml#L23

Also, I don't see these tests being run against the horizontal scaling branch, maybe we should include these being run there now that they're more stable? What do you think?